src/interface/src/app/account-validation/account-validation.component.scss

```diff
@@ -0,0 +1,50 @@
+.validation-root {
```

Comment:
you can avoid this element on your template and use `:host` instead for styling (it will style the angular component tag)